android/src/main/java/com/stripeidentityreactnative/StripeIdentityReactNativeModule.kt

```diff
@@ -11,11 +11,15 @@ class StripeIdentityReactNativeModule(reactContext: ReactApplicationContext) : R
     return "StripeIdentityReactNative"
   }
 
-  lateinit var stripeIdentityVerificationSheetFragment: StripeIdentityVerificationSheetFragment
+  private var stripeIdentityVerificationSheetFragment: StripeIdentityVerificationSheetFragment? = null
+  private var initialized = false
 
   private val mActivityEventListener = object : BaseActivityEventListener() {
     override fun onActivityResult(activity: Activity, requestCode: Int, resultCode: Int, data: Intent?) {
-      stripeIdentityVerificationSheetFragment.activity?.activityResultRegistry?.dispatchResult(requestCode, resultCode, data)
+      if (initialized) {
+        initialized = false
```

Comment:
is there a chance that the identity sheet can be initialized, then presented, canceled, and then presented again without re-initialization? Bc then this `initialized = false` could cause some issues

Could this instead just null check `stripeIdentityVerificationSheetFragment`? AKA:

```
stripeIdentityVerificationSheetFragment?.activity?.activityResultRegistry?.dispatchResult(requestCode, resultCode, data)
```

Reply:
`presentIdentityVerificationSheet` is not directly available to client, we always [call](https://github.com/stripe/stripe-identity-react-native/blob/1fd9bb228de9887b89d003b683f312dc289b9626/src/functions.ts#L10
) `initIdentityVerificationSheet`  before `presentIdentityVerificationSheet`, so it's guaranteeded that a fragment could not be presented without re-iniitalization.

The boolean check here is to make sure that an already used `stripeIdentityVerificationSheetFragment` is not called again if `onActivityResult` is triggered. I was trying to nullify `stripeIdentityVerificationSheetFragment` each time after it's used, but we need to keep its reference to remove it from activity next time `initIdentityVerificationSheet` is called.